cloudbuild.yaml

```diff
@@ -49,6 +70,8 @@ steps:
   waitFor:
   - Initialize git
   - Get Kubernetes Credentials
+  - Copy Kubernetes Credentials
```

Comment:
optional: update this file to use yaml references

Reply:
For a follow up.